campaignion_donation/components/donation_amount.php

```diff
@@ -0,0 +1,213 @@
+<?php
+
+/**
+ * @file
+ * Donation amount webform component.
+ */
+
+/**
+ * Implements _webform_defaults_component().
+ */
+function _webform_defaults_donation_amount() {
+  $defaults = webform_component_invoke('number', 'defaults');
+  unset($defaults['type']);
+  $defaults['extra'] += [
+    'options' => [],
+    'other_option' => TRUE,
+    'other_text' => t('Other'),
```

Comment:
Why not "Other amount"? Seems a better fit as it can only be another amount, not another "something".